src/lwip/standalone/TapInterface.c

```diff
@@ -458,6 +458,26 @@ TapInterface_Init(TapInterface *tapif, const char *interfaceName, u8_t *macAddr)
         perror("TapInterface: ioctl(TUNSETIFF) failed");
         return ERR_IF;
     }
+
+#if 0
```

Comment:
Please feature-flag this according rather than #if 0ing it.

Reply:
I will remove this.  I thought this was a handy note about how to fully mirror interface configuration into LwIP, but at this time it feels like the TAP interface (or perhaps the ethernet bridge) is far more touchy about the MAC addresses than I expected.  That's why I kept the potentially dodgy address assignment based on the PID -- bizarrely enough, it worked better than a fully random 48 bit address.

Reply:
done